src/main/webapp/resources/js/pages/analysis/components/AnalysisOutputFiles.jsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
-import { Table, Icon } from "antd";
+import { Typography } from "antd";
 
-const expandedRowRender = record => <p>{record.description}</p>;
-const showHeader = true;
-const footer = undefined;
+const { Title, Paragraph } = Typography;
 
 export function AnalysisOutputFiles() {
-  return <></>;
+  return (
+    <>
```

Comment:
Don't need these empty brackets

Reply:
Fixed in dc1a014b789d7270dd33c9e420a21a122a5e796a